packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1770,9 +1776,16 @@ function tap<T>(x: T, fn: (x: T) => void): T {
 
 class ImportedSubnet extends Resource implements ISubnet, IPublicSubnet, IPrivateSubnet {
   public readonly internetConnectivityEstablished: IDependable = new ConcreteDependable();
-  public readonly availabilityZone: string;
+  public get availabilityZone(): string {
```

Comment:
I think the linter will complain that this method cannot be declared here. Please move it down between the other (public) methods.

Reply:
moved it below with public method